skynet/modules/stt/streaming_whisper/connection_manager.py

```diff
@@ -14,9 +15,14 @@
 
 class ConnectionManager:
     connections: dict[str, MeetingConnection]
+    flush_after_ms: int
 
     def __init__(self):
         self.connections: dict[str, MeetingConnection] = {}
+        self.flush_after_ms = 2000
+        log.info('Starting the flush audio worker')
+        loop = asyncio.get_running_loop()
+        loop.create_task(self.flush_working_audio_worker())
```

Comment:
I think it would be best to create the task lazily in the `connect` method, if it didn't already exist.